packages/storybook-helpers/main.ts

```diff
@@ -7,7 +7,6 @@ const path = require("path");
 
 const HydrogenPlugin = require("hydrogen-webpack-plugin");
 const TsTransformer = require("@formatjs/ts-transformer");
-const PreloadWebpackPlugin = require("@vue/preload-webpack-plugin");
```

Comment:
You'll want to remove `vue/preload-webpack-plugin` from `/packages/storybook-helpers/package.json` and re-run `pnpm install` to regenerate the lockfile.

Reply:
Weird, I totally did do this :confounded: Maybe a merge conflict resolution gone wrong :sweat_smile: 

Anyway, removed in [remove preload plugin deps](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10048/commits/f9b0744a099bff5708f31b2fddd96bea427db213)